homeassistant/components/shelly/entity.py

```diff
@@ -20,6 +20,27 @@ def temperature_unit(block_info: dict) -> str:
     return TEMP_CELSIUS
 
 
+def shelly_naming(
+    self, wrapper: ShellyDeviceWrapper, block, entity_type: str, channels: int
```

Comment:
We don't need `wrapper` if we use `self.wrapper` later.

Reply:
Thx, done.